### ISSUE TITLE
New update broke something

### ISSUE THREAD

**Issue by Nelon3** [jac3km4/crony-gui#3]
log:

ERROR [crony_gui::host] Unsupported game version (1651007840), exiting!

**Comment by jac3km4** [comment]
yes, this lib will need to be updated, I'll look into it when I have some time

**Comment by Aztec2012** [comment]
> yes, this lib will need to be updated, I'll look into it when I have some time

Any news? All ok?
